ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -67,6 +128,22 @@ const ListingCard = (props: ListingCardProps) => {
             {t("t.seeDetails")}
           </LinkButton>
         )}
+        <Button
```

Comment:
Given that this is pretty visible, I think it's worth it to consolidate these buttons into the single heart icon button, like it is in the mocks. Either that, or we'll want to wait until after the meeting with the mayor to merge this.

Reply:
You probably want to guard showing this based on the existence of the `profile` variable:

```
{profile && <Button...
```

Reply:
Great suggestion.  Updated.